src/API/index.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios'
+
+const get = async(apiUrl: string) => {
+  try {
+    return await axios.get(apiUrl)
```

Comment:
No need of await here. When we return a promise from a function then `await` does not wait for the promise to resolve. Instead of resolving it jut returns an unresolved promise from the function.